index.html

```diff
@@ -395,6 +396,35 @@
       </ul>
     </section>
   </section>
+  <section id="privacy" class="informative">
+    <h2>Privacy and Security</h2>
+    <p>Preload is a declarative fetch primitive that initiates early fetch of
+    resources and separates fetching from resource execution. In effect, it is
+    conceptually similar to initiating a scripted fetch for a resource, but
+    with additional constraints and benefits:</p>
+    <ul>
+      <li>The resource declaration is visible via HTTP headers or HTML markup,
+      which allows the user agent to initiate the fetch earlier, and without
+      blocking on JavaScript execution.</li>
+      <li>The response is not executed and there is no callback to retrieve its
+      payload. The response is retrieved and executed by initiating another
+      fetch (via script, DOM element, etc) that matches the preloaded
+      response.</li>
+      <li>The developer can specify the target context that will consume the
+      response via `as` attribute, which allows the user agent to enforce the
+      relevant CSP policies when initiating the preload fetch. If `as` is
+      omitted, preload defaults to same security and privacy processing as a
+      call to `fetch()` - i.e. subject to `connect-src`.</li>
+    </ul>
+    <p>The site authors are encouraged to take the necessary precautions and
```

Comment:
Wouldn't preload still apply in the case of say, an email inbox preloading subresources (such as email images) before the email is loaded? Or for displaying previews and such in-page?

I think the overall goal being that preloading resources - whether through the existing methods (like XHR or Fetch or CSS) or this declaration - ends up revealing more information about the overall 'activity' than just the page load. This has been admittedly true for over a decade, but perhaps just worth highlighting. Of course, the fact that it is so universally true may be reason enough *not* to mention it.